cmd/cdi-controller/controller.go

```diff
@@ -287,6 +289,12 @@ func start() {
 
 	klog.V(1).Infoln("created cdi controllers")
 
+	// Setup Collectors
+	factory := informers.NewSharedInformerFactory(client, 0)
+	pvcInformer := factory.Core().V1().PersistentVolumeClaims().Informer()
+	pvInformer := factory.Core().V1().PersistentVolumes().Informer()
+	monitoring.SetupPersistentVolumesCollector(pvcInformer, pvInformer)
```

Comment:
You should be able to use the controller runtime cached client, which should be usable once the manager is created
(mgr.GetClient())